releases/v20.2.11.md

```diff
@@ -0,0 +1,128 @@
+- Improved transaction cleanup for disconnected clients to reduce intent buildup. [#65384][#65384]
+- [Scheduled backups](../v20.2/manage-a-backup-schedule.html) with [interleaved tables](../v20.2/interleave-in-parent.html) can now be created with the `include_deprecated_interleaves` option. [#65730][#65730]
+- Calling [`get_bit` or `set_bit`](../v20.2/functions-and-operators.html#string-and-byte-functions) on a byte array argument now goes to the correct index of the underlying bitstring, which matches the Postgres behavior. [#65787][#65787]
+- Previously, [`ALTER DATABASE ... CONVERT TO SCHEMA`](../v20.2/alter-database.html) could potentially leave the schema with invalid privileges thus causing the privilege descriptor to be invalid. [#65813][#65813]
+- Fixed a scenario in which a rapid sequence of splits could trigger a storm of Raft snapshots. This would be accompanied by log messages of the form "would have dropped incoming MsgApp, but allowing due to ...", which tended to occur as part of [`RESTORE`](../v20.2/restore.html)/[`IMPORT`](../v20.2/import.html) operations. [#65500][#65500]
+- Previously, a schema's privilege descriptor could become corrupted upon executing [`ALTER DATABASE ... CONVERT TO SCHEMA`](../v20.2/alter-database.html). This is due to privileges that are invalid on a schema being copied over to the schema, rendering the schema unusable due to invalid privileges. [#65999][#65999]
+- Previously, [`TRUNCATE`](../v20.2/truncate.html) transactionally scanned the meta range to find ranges to unsplit, creating contention with `TRUNCATE`'s new behavior of preserving splits. [#65942][#65942]
+- Fixed the error classification for duplicate index names where the later index was a [`UNIQUE`](../v20.2/unique.html) index. [#64002][#64002]
+- Fixed the error classification for [`ALTER TABLE ... ADD CONSTRAINT ... UNIQUE`](../v20.2/alter-table.html) with the same name as an existing index. [#64002][#64002]
+- Fixed a bug in [`SHOW RANGES`](../v20.2/show-ranges.html) that misattributed localities to nodes when using multiple stores. [#66038][#66038]
+- Fixed a bug where [incremental backups](../v20.2/take-full-and-incremental-backups.html) with [`revision_history`](../v20.2/backup.html#options) would sometimes fail if a view was altered between incremental backups. [#66072][#66072]
+
+### Performance improvements
+
+- Fixed an issue where the [storage layer](../v20.2/architecture/storage-layer.html) would under some rare write-heavy workloads start off by writing SSTable files that were too small to manage effectively. The storage layer now creates fewer, larger files under those workloads, in line with expected behavior. [#65481][#65481]
+- Fixed an issue in the optimizer that prevented spatial predicates of the form `(column && value) = true` from being index-accelerated. These queries can now use a [spatial index](../v20.2/spatial-indexes.html), if one is available. [#65988][#65988]
```

Comment:
From reading the original issue, this seems like a bug, but the effect of fixing it is a performance improvement, so it's hard to classify. I lean 'bug fix' because it did not use the index, as we would have intended. If you feel that it's worth reconsidering, please see the original issue https://github.com/cockroachdb/cockroach/issues/65684 and feel free to make a call or ask Ian for an opinion.

Reply:
Yes, it is a bug and the engineer confirmed this and that it made it more difficult for _CockroachDB_ to manage effectively with smaller files.